### PR TITLE
updated atoms.coordinates() with atoms.positions #3467

### LINKED ISSUE #3467: replace atoms.coordinates() with atoms.positions
Since 2.0, atoms does not have coordinates() anymore and only positions is supported.

- [ ] Needs to be fixed:
https://github.com/MDAnalysis/mdanalysis/blob/46c8badd8b8967dc25f8125ef87582490ab497e3/package/MDAnalysis/analysis/encore/covariance.py#L218
- [ ] Also, fix benchmarks that still use coordinates() (but this could be made another PR)

### PR DESCRIPTION
**Pull request by manishsaini6421**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/analysis/encore/covariance.py
Comment (lilyminium) [inline package/MDAnalysis/analysis/encore/covariance.py:218]: Could you please add a test that check this fix works?

**Review by lilyminium — CHANGES_REQUESTED** [review]

**Comment by orbeckst** [comment]
@manishsaini6421 please be aware that there's another PR #3479 open which would take precedence over yours. If we don't see any further progress on the other PR within 4 days we will [close it as stale](https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-FAQ#closing-stale-prs) on 2022-04-01 and then your PR will be next in line.

See https://github.com/MDAnalysis/mdanalysis/pull/3479#issuecomment-1081305333 on the other PR.

Thank you for your patience.

**Comment by hmacdope** [comment]
@manishsaini6421 there are several other instances of  `.coordinates` to fix, see the original issue #3467 for more info.

**Review by hmacdope — CHANGES_REQUESTED** [review]
See my comment above

**Comment by manishsaini6421** [comment]
@hmacdope Sir,Thanks for your support i'll do it now.

File: package/MDAnalysis/analysis/encore/covariance.py
Comment (manishsaini6421) [reply package/MDAnalysis/analysis/encore/covariance.py:218]: can you tell me please, how to add test?

**Comment by orbeckst** [comment]
Hi everyone, why was this PR closed?

**Comment by manishsaini6421** [comment]
why it is closed?, and my pull request is also under review till now.

EDIT: see https://github.com/MDAnalysis/mdanalysis/pull/3577#issuecomment-1082132949 — please continue work on *this* PR.

**Comment by manishsaini6421** [comment]
@orbeckst Sir, please check my PR now .

**Comment by orbeckst** [comment]
Re-open this PR (you should see a button "Reopen and comment") and close PR #3577.

**Comment by lilyminium** [comment]
> Hi everyone, why was this PR closed?

I think it could have been automatically closed by GitHub because @manishsaini6421 made a new PR from the same branch. There can only ever be one PR from branch A to branch B.

Edit: maybe not, since #3577 stays open...

File: package/MDAnalysis/analysis/encore/covariance.py
Comment (lilyminium) [reply package/MDAnalysis/analysis/encore/covariance.py:218]: @manishsaini6421 please write a function with code that uses this pathway. It should fail with the original `.coordinates()` and succeed with the new `.positions`. Have a look at the tests in https://github.com/MDAnalysis/mdanalysis/blob/develop/testsuite/MDAnalysisTests/analysis/test_encore.py and more generally throughout https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite/MDAnalysisTests for ideas :)

**Comment by orbeckst** [comment]
@manishsaini6421 I am slightly confused why your branch [manishsaini6421:shiny-new-feature](https://github.com/manishsaini6421/mdanalysis/tree/shiny-new-feature) only contains 1 change in this PR but contained 2 changes in the other closer PR #3577. In any case, can you please check that you are on the current branch:
```
git branch --show-current
```
should show **shiny-new-feature**.

Then make sure you have everything from GitHub
```
git pull
```
Then say
```
git status
```
Do you see any changes that are not committed? If so add the changes, commit, and push.

In particular, make the changes that I can see in the closed PR https://github.com/MDAnalysis/mdanalysis/pull/3577/files (in [benchmarks/benchmarks/analysis/rms.py](https://github.com/MDAnalysis/mdanalysis/pull/3577/files#diff-115cad587a5c523f43987c11ba4d46ed6fa91f8d1c43946ddc531fa94d87615e)).

Check that the changes show up in _this_ PR. If not, let us know.

**Comment by manishsaini6421** [comment]
@orbeckst Sir, there  is lot of confusion, please can you fix a meeting so i can do work efficiently.

**Comment by IAlibay** [comment]
> @orbeckst Sir, there is lot of confusion, please can you fix a meeting so i can do work efficiently.

@manishsaini6421 I'll let the other @MDAnalysis/gsoc-mentors weigh in here, but we generally don't have sufficient time to have focused meetings with individual applicants during the GSoC application period. The mentors all have separate jobs outside of MDAnalysis, usually at academic institutions where teaching is currently at its peak.

I would instead suggest that you ask your questions on the mailing list or on discord, where others may get to asynchronously answer your questions.

**Comment by hmacdope** [comment]
@manishsaini6421 part of the issue appears to be that you are remaking, deleting or altering the `shiny-new-feature` branch repeatedly as it no longer contains any of the commits in this PR.

**Comment by orbeckst** [comment]
@manishsaini6421 as a starting point for debugging your issue, please answer my questions in https://github.com/MDAnalysis/mdanalysis/pull/3576#issuecomment-1082432890

**Comment by manishsaini6421** [comment]
@orbeckst Sir, this is because of creating 2nd PR.

**Comment by orbeckst** [comment]
This PR is now considered the primary PR for issue #3467 because PR #3479 was closed as stale. Nevertheless, the old PR contains useful information so please consult it.

**Comment by orbeckst** [comment]
@manishsaini6421 are you still working on this PR?

If not we will close it in 4d as stale.

**Comment by orbeckst** [comment]
I read https://github.com/MDAnalysis/mdanalysis/pull/3579#issuecomment-1083163254 and it looks as if it's not easily possible for @manishsaini6421 to continue working on this PR because their PR #3579 also uses the same branch name "shiny-new-feature". I think there's some misunderstanding for how branching with git works. In any case, I am **closing this PR** because it's not going to be easy for the contributor to continue.

@manishsaini6421 please open a new PR and use a **distinct branch name** (such as "issue-3467-replace-coordinates").

**Comment by manishsaini6421** [comment]
@orbeckst ok sir i'll do it but first of all review reqired on #3579 then i can create new PR with another name.

**Comment by manishsaini6421** [comment]
@orbeckst sir can you review it please.

**Comment by orbeckst** [comment]
We want you to do one PR after another. Focus on #3579 ; given that someone is already reviewing it, you just have to wait on the reviews there. This can take some time and we cannot drop other work as soon as there are changes on a PR; it can take a day so be patient, please.

Once PR #3579 is merged you can consider opening a new PR for issue #3467 . We are much more impressed with applicants who efficiently complete one problem than ones that open on multiple problems at the same time. Therefore, we are not going to review this closed PR.

**Comment by manishsaini6421** [comment]
@orbeckst In #3579  i have made some changes after review according to changes requested but from last 2 days no one is replying on my comments . sir i am waiting for approval ,can you check it once please.

**Pull request closed without merging**
